odim-controller/scripts/kube_deploy_nodes.yaml.tmpl

```diff
@@ -12,11 +12,11 @@
 #License for the specific language governing permissions and limitations
 #under the License.
 
-deploymentID: <Unique identifier for the deployment>
-httpProxy: <HTTP Proxy to be set in the nodes>
-httpsProxy: <HTTPS Proxy to be set in the nodes>
-noProxy: <NO PROXY env to be set in the nodes>
-nodePasswordFilePath: <Absolute path of the file containing encrypted node password>
+deploymentID:
```

Comment:
Better to keep the description of parameters

Reply:
Yeah, but README and GSG has more detailed description on the parameters, and here it's kind of incomplete, and wasn't present for all params, so removed it.